samples/dnn/audio_visual_speech_recognition.py

```diff
@@ -0,0 +1,239 @@
+from collections import deque
+import cv2
+import numpy as np
+import argparse
+
+'''
+AVSpeechRecognition
+
+    How to obtain the model required for this sample:
+    Option 1: Download the model from https://drive.google.com/file/d/1xuwk5ZQagKFoTXev27zvlSg8dAmSJoo7/view?usp=sharing
+    Option 2: Convert the model using pretrained torch model and base repo.
+        Use the colab notebook here: https://colab.research.google.com/drive/1awBCZ5O6uAT32cHvufNWad5m6q26TuqQ?usp=sharing
```

Comment:
Please refactor notebook:
1. in the notebook  should be only the parts related to model conversion
2. don't copypaste code from original repo, you can import necessary classes and functions. For example,
```python
from lipreading import model
from lipreading import utils
import torch.onnx

with open('configs/lrw_resnet18_mstcn.json') as fp:
    config = json.load(fp)


class CustomLipreading(model.Lipreading):
  def forward(self, x):
     lengths = [20]
     return model.Lipreading.forward(self, x, lengths)

tcn_options = {
    'num_layers': config['tcn_num_layers'],
    'kernel_size': config['tcn_kernel_size'],
    'dropout': config['tcn_dropout'],
    'dwpw': config['tcn_dwpw'],
    'width_mult': config['tcn_width_mult'],
}

audio_model = model.Lipreading(
    num_classes=500,
    tcn_options=tcn_options,
    backbone_type=config['backbone_type'],
    relu_type=config['relu_type'],
    width_mult=config['width_mult'],
    extract_feats=False,
    modality = 'raw_audio'
)

utils.load_model(Path('models/lrw_resnet18_mstcn_audio_adamw.pth.tar'), audio_model)
audio_model.to('cpu')
audio_model.eval()

audio_input = torch.randn(1,1,20*640)
torch.onnx.export(audio_model, audio_input, 'lipreading_audio.onnx' , opset_version=11)
torch.onnx.export(audio_model, (audio_input, [20]), 'lipreading_audio.onnx' , opset_version=11)
```

Reply:
@spazewalker please remove link to notebook